control-plane/pkg/reconciler/testing/objects_channel.go

```diff
@@ -259,6 +259,8 @@ func ChannelReference() *contract.Reference {
 		Uuid:      ChannelUUID,
 		Namespace: ChannelNamespace,
 		Name:      ChannelName,
+		Kind:      "KafkaChannel",
+		Version:   messagingv1beta1.SchemeGroupVersion.String(),
```

Comment:
I think `Version` here is meant to be `ResourceVersion` as opposed to `GroupVersion`

Reply:
Oh okay, I need the group version so I will add a new field for that then. Thanks!